### PR TITLE
add AcalaAddress and KaruraAddress

### PR DESCRIPTION
**Pull request by zjb0807**
Closes: #93

Relates: #107  `export const FA_USDT: "0x0000000000000000000500000000000000000007";`

### CODE REVIEW

**Review by xlc — COMMENTED** [review]
This moves some file location so needs a version bump and ensure readme and other docs are updated

**Comment by zjb0807** [comment]
> This moves some file location so needs a version bump and ensure readme and other docs are updated

I'll publish it in another PR

**Comment by zjb0807** [comment]
@ThunderDeliverer  Update the docs after the new version.
https://github.com/AcalaNetwork/hardhat-tutorials/blob/master/precompiled-token/README.md

**Comment by ThunderDeliverer** [comment]
> @ThunderDeliverer Update the docs after the new version. https://github.com/AcalaNetwork/hardhat-tutorials/blob/master/precompiled-token/README.md

@zjb0807 I'll update the docs once the PR is merged, but I'll have a doc change request ready to merge as soon as it does 👌

**Review by gluneau — APPROVED** [review]
LGTM

File: contracts/utils/AcalaAddress.sol [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	address public constant DOT = 0x0000000000000000000100000000000000000002;
+	address public constant LDOT = 0x0000000000000000000100000000000000000003;
+	address public constant TAP = 0x0000000000000000000100000000000000000004;
+	address public constant RENBTC = 0x0000000000000000000100000000000000000014;
+	address public constant CASH = 0x0000000000000000000100000000000000000015;
+	address public constant LCDOT = 0x000000000000000000040000000000000000000d;
+	address public constant SA_DOT = 0x0000000000000000000300000000000000000000;
+	address public constant SA_3USD = 0x0000000000000000000300000000000000000001;
+	address public constant LP_ACA_AUSD = 0x0000000000000000000200000000000000000001;
+	address public constant LP_DOT_AUSD = 0x0000000000000000000200000000020000000001;
```
[the remark this answers is not in the document]
Comment (zjb0807) [reply contracts/utils/AcalaAddress.sol:17]: good point. I'll fix it

**Pull request merged**